<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addGraveDialog</class>
 <widget class="QDialog" name="addGraveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 77, 56);</string>
  </property>
  <widget class="QLineEdit" name="graveNoLineEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationLineEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightLineEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 77, 56);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Add Grave</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 77, 56);
font: 14pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 77, 56);
font: italic 22pt &quot;Segoe UI&quot;;
font: 600 22pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Add Grave</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Grave No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Grave Height:</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>graveNoLineEdit</zorder>
  <zorder>locationLineEdit</zorder>
  <zorder>heightLineEdit</zorder>
  <zorder>addButton</zorder>
  <zorder>backButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
